app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,99 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```

Comment:
Please combine `new` and `edit` page in single file. If you don't know how and why we have to do this. I'll explain.